### ISSUE TITLE
High cpu usage when floating window of UG NX is present

### ISSUE THREAD

**Issue by QiangF** [ch11ng/exwm#586]
My exwm version is 0.22.1. The follwing is the debug output:
https://gist.github.com/QiangF/64e8cccc86a919a3e0de59b5c4fedf1d

**Comment by ch11ng** [comment]
With `sort gistfile1.txt | uniq -c | sort -gr` it seems handling a property called `_MOTIF_DRAG_RECEIVER_INFO` took majority of the time.

```
    172 exwm-input--on-PropertyNotify:  
    151 exwm--on-PropertyNotify:        Unhandled: _MOTIF_DRAG_RECEIVER_INFO(421)
    151 exwm--on-PropertyNotify:        atom=_MOTIF_DRAG_RECEIVER_INFO(421)
     24 exwm--on-ClientMessage: atom=_NET_WM_STATE(327)
     16 exwm-xim--on-request:   
     16 exwm-xim--on-ClientMessage:     
[...]
```

Could you confirm whether there is such message without floating window?

**Comment by QiangF** [comment]
I can confirm _MOTIF_DRAG_RECEIVER_INFO doesn't appear without floating window.

**Comment by ch11ng** [comment]
I'm not sure if this can be disabled on the application side, or if this is because some wrongdoing on our side that makes the application keeps sending this message.  It's not a standard property anyway, so I guess it can be ignored.

**Comment by QiangF** [comment]
Yes, it should be ignored. I am sure I have nothing to do with it.

**Comment by QiangF** [comment]
Looks like it's the sender (ug nx) that are using the cpu, then there's nothing exwm can do.

**Comment by ch11ng** [comment]
> Is it even possible to use two window manager in a single X server?

No.

Does this application has single-window mode like recent versions of GIMP?
